gallery/webpack.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = function(env = { production: false }) {
     entry: './main.tsx',
     output: {
       filename: 'bundle.js',
-      publicPath: '/'
+      publicPath: ''
```

Comment:
Usually `./` is the way to go here

Reply:
> Usually `./` is the way to go here

Changed it here: d06c6ce171050742d5d82b746fb11965a9f1c55e